scene/main/node.cpp

```diff
@@ -3088,6 +3088,21 @@ void Node::set_display_folded(bool p_folded) {
 	data.display_folded = p_folded;
 }
 
+void Node::set_display_folded_bind(bool p_folded) {
+	ERR_THREAD_GUARD
+
+	if (p_folded == data.display_folded) {
+		return;
+	}
+	set_display_folded(p_folded);
```

Comment:
Should this set `data.display_folded` directly to avoid getting another `ERR_THREAD_GUARD`? Or it doesn't matter?

Reply:
I called `set_display_folded()` to avoid code duplication, but I guess it doesn't make sense. I could restructure the code, but the equality check isn't needed in the base method.